PR_REVIEW_CHECKLIST.md

```diff
@@ -0,0 +1,98 @@
+    - Ex: if the issue was to prevent the user from saving a field when it is empty, besides testing that we should also test that they can save the field _if it is not empty_.
+
+# Testing the changes
+
+1. Make sure there are no console errors related to the PR. If you find console errors in a branch you’re reviewing, check if they also exist in `main`.
+    1. If they do, proceed as normal (report them in #expensify-open-source if you think they need to be fixed).
+    2. If the errors do not exist in `main`, report it to the PR author - they need to be fixed before the PR can be merged.
+2. Test the changes on **all platforms**. Follow this guide (needs link) to set up each platform.
+    - If you’re unable to boot a platform for any reason, ask for help in the #expensify-open-source Slack channel. Your issue might already have been addressed there, so be sure to search first.
+3. Should we add regression testing for code introduced in this PR?
+    - If the PR introduces **new** pages, features, etc. then we **should** have new regression tests written. Please note this in your PR so that Applause can make sure it gets added.
+    - If the PR fixes a bug, Applause will check to make sure the existing regression testing steps already cover the workflow fixed by the bug.
+
+# Reviewing the code
+
+## Good code patterns to require
+
+1. Check that functions have comments when appropriate.
+    - If the function has params or returns something, use JSDocs syntax to describe them.
+        - Indicate the param name(s), data type passed and / or returned, and purpose if not immediately obvious.
+    - Obvious functions (with no params / return value) should not have comments.
+    - **In short: _Add comments & docs often, only when useful._**
+2. All copy / text shown to users in the product should be stored in the `src/languages/*` files for localization.
+3. Platform-specific files should follow the proper naming convention (see [Platform-Specific File Extensions](https://github.com/expensify/app#platform-specific-file-extensions))
+    - Example: `MyComponent/index.android.js`
+
+## Code patterns to check for improvement
+
+1. Is there appropriate error handling?
+2. Reuse...
+3. Are there any changes that could've been applied to one component that were applied to every usage of the component instead?
+    - Example: https://github.com/Expensify/App/pull/3129#pullrequestreview-668271068
+4. Ensure fixes for just android or ios aren't put in a `index.native.js`. Instead they should be in a specific `index.ios.js` or `index.android.js` file.
+    - Put another way, ensure platform-specific solutions are not applied cross-platform.
+5. Look for usages of react fragment wrappings and confirm they are necessary.
+    - Also look for any unnecessary `View`s and think if they should be replaced with React fragments.
+6. Were any new styles added? Can we reuse existing styles instead? Can we generalize the new style?
```

Comment:
Let's link to the `STYLING.md` doc here rather than ask all these questions just say

```
6. Were any new styles added? Are we following the guidelines in [`STYLING.md`](..linktoit)?
```

Reply:
Agreed, I like this